client/src/main/java/gui/controllers/PlacementSceneController.java

```diff
@@ -126,6 +126,7 @@ private void processMessagesFromServer() {
             @Override
             protected String call() {
                 String serverMessage = serverConnection.getMessage();
+                System.out.println(serverMessage);
```

Comment:
use logger pls

Reply:
We will add it later on. This issue will be tracked under number #37